registry.json

```diff
@@ -5564,5 +5564,20 @@
     "telegram": "tabbycatsg",
     "operatorName": "ODYNG",
     "github": "odyng"
+  },
+  {
+    "ticker": "PFL01",
+    "poolId": "24bfb62716d755bbf035fcb3f7e87a8d325fde0f04062e692aff3216",
+    "social": {
+      "website": "https://poolforlovelace.me",
+      "twitter": "PFL_pool",
+      "facebook": "",
+      "discord": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "PFL01",
```

Comment:
Please put the telegram username of the SPO and not a group or channel

Reply:
You need to put the telegram username of an individual in the "telegram" field, not the group or channel.